howso/ablation.amlg

```diff
@@ -194,22 +204,34 @@
 	; influence_weight_entropy_threshold: optional, default !autoAblationInfluenceWeightEntropyThreshold. cases with
 	; 	influence weight entropy above this quantile will be removed
 	; weight_feature: optional, default !autoAblationWeightFeature. name of feature whose values to use as case weights
+	; cache_value: flag, default true.  when true will cache the computed value
 	#!MaxInfluenceWeightEntropyToKeepQuery
 	(declare
 		(assoc
 			influence_weight_entropy_threshold !autoAblationInfluenceWeightEntropyThreshold
 			weight_feature !autoAblationWeightFeature
+			cache_value (true)
 		)
 
-		(compute_on_contained_entities (list
-			(query_exists !internalLabelInfluenceWeightEntropy)
-			(query_not_equals !internalLabelInfluenceWeightEntropy 0)
-			(query_quantile
-				!internalLabelInfluenceWeightEntropy
-				influence_weight_entropy_threshold
-				weight_feature
-			)
+		(declare (assoc
```

Comment:
I think we may want to wrap this (compute_on_contained_entities) in some logic to check if its (null) or something like that. I think I remember Jacob ran into an issue where this ran when there were no cases that had the influence weight entropy cached or something like that. 

Just ensure that this value is usable, even if the model doesn't have the data for it.

Reply:
oh it looks like this might be covered after the method call.. though the Trainee attribute could get the sketchy value